### DISCUSSION TITLE
dac set clock frequency

### DISCUSSION THREAD

**Discussion by gerthalberg** [schreibfaul1/ESP32-audioI2S#155] [Q&A]
Hi I'm trying to get this up and running with this Audio library and the Cellie WM8978 library. In the example code there is a call dac.setPinClockFreq(I2S_MCLKPIN, I2S_MFREQ);. however the wm8978 library does not implement this this method - can it be omitted or is there a misunderstanding between the two libraries.

**Comment by CelliesProjects** [comment]
The example is outdated and the correct call is
`i2s_mclk_pin_select(const uint8_t pin)` 

See https://github.com/schreibfaul1/ESP32-audioI2S/blob/master/src/Audio.h#L155

Thanks for reporting  this.

**Comment by CelliesProjects** [comment]
Commit #156 should fix this issue.

The example should compile correctly with that commit merged.

For clarity, the call has moved from the 'WM8978' to the 'ESP32-audioI2S' library.

So `dac.setPinClockFreq(I2S_MCLKPIN, I2S_MFREQ)` should be changed to `audio.i2s_mclk_pin_select(I2S_MCLKPIN)`.

**Reply by gerthalberg** [reply]
Thanks a lot both of You.
